ros/src/tl_detector/light_classification/ssd_tl_classifier.py

```diff
@@ -5,25 +5,38 @@
 import tensorflow as tf
 import numpy as np
 import cv2
+from PIL import Image, ImageFont, ImageDraw
 
 from abc import ABCMeta, abstractmethod
 
 from styx_msgs.msg import TrafficLight
 from light_classification.tl_classifier import TLClassifier
 
-
 class SSDTLClassifier(TLClassifier):
 
     __metaclass__ = ABCMeta
 
-    def get_state_count_threshold(self, last_state):
+    @staticmethod
+    def get_state_count_threshold(last_state):
         if last_state == TrafficLight.RED:
             # High threshold for accelerating
             return 3
-
         # Low threshold for stopping
         return 1
 
+    @staticmethod
+    def convert_box_coords(boxes, height, width):
+        """
+        Converts bounding boxes from normalized
+        coordinates (0 to 1), to image coordinates
+        """
+        box_coords = np.zeros_like(boxes)
+        box_coords[:, 0] = boxes[:, 0] * height
+        box_coords[:, 1] = boxes[:, 1] * width
+        box_coords[:, 2] = boxes[:, 2] * height
+        box_coords[:, 3] = boxes[:, 3] * width
+        return box_coords
+
     @staticmethod
     def load_graph(graph_file):
```

Comment:
Since we use so-called "protected" methods in classifier classes, it is reasonable to make this method also protected: `_load_graph`.